src/main/webapp/app/entities/share/share-visualization.scss

```diff
@@ -0,0 +1,4 @@
+.visualization-share {
+  height: calc(100vh - 100px);
+  min-height: 100%;
+}
```

Comment:
can check whether u can use these css as View props
for example <View minHeight="100%"></View>

Reply:
this one is div element not View

Reply:
i know..can you replace div with View component

Reply:
done

Reply:
if done then there is no use of this src/main/webapp/app/entities/share/share-visualization.scss